arch/x86/kvm/pkvm/cpu.c

```diff
@@ -21,6 +21,7 @@ struct cpumask __cpu_possible_mask __ro_after_init;
 unsigned int nr_cpu_ids;
 DEFINE_PER_CPU(u64, x86_spec_ctrl_current);
 DEFINE_STATIC_KEY_FALSE(switch_vcpu_ibpb);
+unsigned int tsc_khz;
```

Comment:
Is it needed, if we're gonna let the host control the TSC for now?

Reply:
No, we can also remove it for now.